### DISCUSSION TITLE
是否可以把下载弹幕单独出来（现在想下载弹幕必须下载视频）

### DISCUSSION THREAD

**Discussion by dongshuyan** [the1812/Bilibili-Evolved#2163] [功能建议 / Ideas]
有没有什么办法可以单独下载弹幕呢？~

**Comment by Geeyun-JY3** [comment]
这不是可以吗：
![image](https://user-images.githubusercontent.com/49396421/130162282-926453d7-617a-4bac-96ba-d65abfb29f39.png)

**Reply by the1812** [reply]
盲猜又是番剧区的
